seaborn/_statistics.py

```diff
@@ -345,3 +345,39 @@ def __call__(self, x1, x2=None, weights=None):
             return self._eval_univariate(x1, weights)
         else:
             return self._eval_bivariate(x1, x2, weights)
+
+
+class ECDF:
+
+    def __init__(self, stat="proportion"):
+
+        _check_argument("stat", ["count", "proportion"], stat)
+        self.stat = stat
+
+    # Do we need bivariate ECDF?
+
+    def _eval_univariate(self, x, weights):
```

Comment:
Might a docstring help future contributors to the library?

I thought of the following as a strawman, though I am merely deciphering the implementation intent from the code:

```suggestion
    def _eval_univariate(self, x, weights):
        """
        Return x, y for plotting ECDF.
        
        X-values are the data values,
        while y-values are either the cumulative proportion (if self.stat is "proportion")
        or the cumulative number (if self.stat is "count").
        of values equal to or below that value,

        """
```